docs/reference/query-dsl/filters/ids-filter.asciidoc

```diff
@@ -2,9 +2,7 @@
 === Ids Filter
 
 Filters documents that only have the provided ids. Note, this filter
-does not require the <<mapping-id-field,_id>>
-field to be indexed since it works using the
-<<mapping-uid-field,_uid>> field.
+uses the <<mapping-uid-field,_uid>> field.
```

Comment:
Add "If no `type` is specified, all types defined in the index mapping are tried." (or something like that)